docs/guides/image-optimization/imageurl-image-optimization.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Use Image Optimization with imageUrl
+description: Learn how to migrate an application using Auth.js to use Clerk for authentication.
```

Comment:
The description here looks like it may be leftover from a different guide. Maybe something like: "How to display scaled down user profile images from Clerk"